apps/react/src/challenges/memory-game/components/atoms/Cell.jsx

```diff
@@ -0,0 +1,12 @@
+import './atoms.css';
```

Comment:
need to use it as a module so that styles won't effect other challenges

Reply:
Added module.css